app/models/affiliate.rb

```diff
@@ -19,31 +19,46 @@ class Affiliate < ApplicationRecord
   MAX_NAME_LENGTH = 33.freeze
 
   with_options dependent: :destroy do |assoc|
-    assoc.has_many :affiliate_feature_addition
-    assoc.has_many :affiliate_twitter_settings
-    assoc.has_many :boosted_contents
-    assoc.has_many :connections, -> { order 'connections.position ASC' }
-    assoc.has_many :connected_connections, :foreign_key => :connected_affiliate_id, :source => :connections, :class_name => 'Connection'
-    assoc.has_many :document_collections, -> { order 'document_collections.name ASC, document_collections.id ASC' }
-    assoc.has_many :excluded_domains, -> { order 'domain ASC' }
-    assoc.has_many :excluded_urls
-    assoc.has_many :featured_collections
-    assoc.has_many :features, :through => :affiliate_feature_addition
-    assoc.has_many :flickr_profiles, -> { order 'flickr_profiles.url ASC' }
-    assoc.has_many :i14y_memberships
-    assoc.has_one :image_search_label
-    assoc.has_many :indexed_documents
-    assoc.has_many :memberships
-    assoc.has_many :navigations, -> { order 'navigations.position ASC, navigations.id ASC' }
-    assoc.has_many :routed_queries
-    assoc.has_many :rss_feeds, -> { order 'rss_feeds.name ASC, rss_feeds.id ASC' }, as: :owner
-    assoc.has_many :sayt_suggestions
-    assoc.has_many :site_domains, -> { order 'domain ASC' }, inverse_of: :affiliate
-    assoc.has_one :site_feed_url
-    assoc.has_many :superfresh_urls
-    assoc.has_one :alert
-    assoc.has_many :watchers, -> { order 'name ASC' }
-    assoc.has_many :tag_filters, -> { order 'tag ASC' }
+    assoc.has_many(:affiliate_feature_addition)
```

Comment:
As `has_many` is part of the internal Rails DSL, parentheses should not be used around the args:
https://github.com/rubocop-hq/ruby-style-guide#methods-that-are-part-of-an-internal-dsl

Reply:
ahh right. rubcop was complaining about them.

Reply:
We could add the Rails association methods to the list of ignored methods for that cop:
```
Style/MethodCallWithArgsParentheses:
  Description: 'Use parentheses for method calls with arguments.'
...
  IgnoredMethods: ['require']
```
We could also consider disabling that cop entirely. It's disabled by default because "There are just too many false positives at the moment". See https://github.com/rubocop-hq/rubocop/issues/6564#issuecomment-445700075

Reply:
I think disabling it may be a good idea. And just put parenthesis as needed.  I'll do that now.